.github/workflows/docker-image.yml

```diff
@@ -81,4 +81,4 @@ jobs:
           distribution: 'temurin'
 
       - name: Build and Push the Docker image
-        run: ./dev/docker/build-docker.sh --platform all --type ${image_type} --image ${image_name} --tag ${{ github.event.inputs.tag }} --latest
+        run: sudo rm -rf /usr/local/lib/android && sudo rm -rf /opt/hostedtoolcache/CodeQL && ./dev/docker/build-docker.sh --platform all --type ${image_type} --image ${image_name} --tag ${{ github.event.inputs.tag }} --latest
```

Comment:
What does the directory `/opt/hostedtoolcache/CodeQL` is used for?

Reply:
Git action vm pre-installed some packages. We don't need it. So we can remove it. We don't need to care what is used for . Arrow pipeline also removes this package.

Reply:
Git action vm pre-installed some packages. We don't need it. So we can remove it. We don't need to care what is used for . Arrow pipeline also removes this package.

Reply:
I see.